signin/index.js

```diff
@@ -0,0 +1,78 @@
+const email = document.querySelector('#email');
+const password = document.querySelector('#password');
+const loginBtn = document.querySelector('button');
+const eyeOff = document.querySelector('.eye-off');
+
+function reset() {
+    email.classList.remove('warning');
+    if (document.querySelector('.warning-text') != undefined) {
+        document.querySelector('.warning-text').remove();
+    }
```

Comment:
`!=` !!!
동등 연산자가 들어가 있네요 😭 최대한 지양해주세요!
왜 사용하면 안 되는지, `!==` 와의 차이점은 무엇인지 꼭 학습해주세요!

그리고 코드가 반복되니, 아래와 같이 수정해보면 어떨까요?

```suggestion
    const warningText = document.querySelector('.warning-text');
    if (warningText) {
        warningText.remove();
    }
```